src/main/java/it/gov/pagopa/admissibility/service/commands/operations/DeleteInitiativeServiceImpl.java

```diff
@@ -18,13 +21,17 @@ public class DeleteInitiativeServiceImpl implements DeleteInitiativeService{
     private final OnboardingFamiliesRepository onboardingFamiliesRepository;
     private final AuditUtilities auditUtilities;
     private final OnboardingContextHolderService onboardingContextHolderService;
+    private final String pagination;
+    private final String delayTime;
 
-    public DeleteInitiativeServiceImpl(DroolsRuleRepository droolsRuleRepository, InitiativeCountersRepository initiativeCountersRepository, OnboardingFamiliesRepository onboardingFamiliesRepository, AuditUtilities auditUtilities, OnboardingContextHolderService onboardingContextHolderService) {
+    public DeleteInitiativeServiceImpl(DroolsRuleRepository droolsRuleRepository, InitiativeCountersRepository initiativeCountersRepository, OnboardingFamiliesRepository onboardingFamiliesRepository, AuditUtilities auditUtilities, OnboardingContextHolderService onboardingContextHolderService, @Value("${app.delete.paginationSize}") String pagination, @Value("${app.delete.delayTime}") String delayTime) {
```

Comment:
why are you using String? this values are supposed to be integer, declare them as int!

Reply:
Moreover, let's format this line in order to have a single line per parameter, they are too numerous to let them in single line